textinput_model.go

```diff
@@ -33,133 +33,154 @@ func (p *textinputPlugin) addChar(char []rune) {
 	p.updateEditingState()
 }
 
-func (p *textinputPlugin) MoveCursorHome(modsIsModifier bool, modsIsShift bool, modsIsWordModifierShift bool, modsIsWordModifier bool) {
+func (p *textinputPlugin) moveCursorHomeSimple() {
```

Comment:
What does "Simple" mean?

Reply:
Either you move the cursor to the beginning of the line with or without the text selected.